tests/shell/pipeline/commands/external.rs

```diff
@@ -496,15 +496,15 @@ mod external_command_arguments {
     #[cfg(not(windows))]
```

Comment:
Then we can remove this, it should be able to run on windows too

Reply:
OK, I'll remove it.